content-services/content-sources-frontend
# Fixes 2613: React 18/Patternfly 5/Node 18 upgrade

## Summary

This updates our application to use React 18, Patternfly 5, and node 18. 

## Testing steps

- Ensure if checking locally you are using nvm and doing "nvm use" installing the correct new node version (if necessary)
- npm install
- Run the app as you regularly would. 
- Regression testing is basically what we're looking for.

Additional information: 

- There is a [known bug ](https://issues.redhat.com/browse/HMS-2588) with the popular repository page that stems from the API not returning an associated UUID.  This is confirmed as not UI related.
